src/test/HT_DataService.spec.ts

```diff
@@ -121,6 +121,19 @@ describe("HT_DataService", function () {
         expect(localSpy4set).toHaveBeenCalled();
     });
 
+    it("calls and sets value to local storage when salt is set", function () {
+
+        localSpy4get.and.returnValue(<HT_Data>{"saltBits":"XXX"});
```

Comment:
Is this needed? I don't see it being used anywhere unless I'm missing something?